Game

```diff
@@ -0,0 +1 @@
+what should i do
```

Comment:
Here's one (of many) options for getting started on creating games with Flutter: https://codelabs.developers.google.com/codelabs/flutter-flame-brick-breaker

Hope that helps!